api-spec/sftt/api-spec.md

```diff
@@ -135,6 +135,7 @@ Members of a team have roles specified in the following table:
 | NONE           | 0        |
 | MEMBER         | 1        |
 | LEADER         | 2        |
+| PENDING        | 3        |
```

Comment:
What kind of view of the team would a pending member have? The same as none, but with a line saying something like "You have requested to join this team, your application is pending"?

Reply:
Exactly